contrib/python/podman/test/test_images.py

```diff
@@ -64,7 +64,7 @@ def test_create(self):
         self.assertEqual(actual.status, 'configured')
 
         ctnr = actual.start()
-        self.assertIn(ctnr.status, ['running', 'exited'])
+        self.assertIn(ctnr.status, ['running', 'stopped', 'exited'])
```

Comment:
Did this turn out to be a valid status compared to what we chatted about the other day?

IMHO, this test should probably only check for `running`, and the container should execute 'sleep 1h'.  That's would unambiguously confirm the image can be started.  Otherwise, there could be a failure leading to one of the other statuses, and we wouldn't know :confused:

Reply:
Ugg, sorry, I'm discombobulated today, I owe you a BZ number don't I?

Reply:
Issue #1643 opened for this.